src/proto/beats.proto

```diff
@@ -10,22 +11,75 @@ enum Source {
   USER = 2;
 }
 
-message Beat {
-  optional int32 frame_position = 1;
-  optional bool enabled = 2 [ default = true ];
-  optional Source source = 3 [ default = ANALYZER ];
+enum SectionType {
+  INTRO = 0;
+  OUTRO = 1;
+  OTHER = 2;
+}
+
+// Setting grey as the default color
```

Comment:
Why do we need to specify a default color at the protocol level? This is an arbitrary decision and doesn't belong here. Implicitly using (0, 0, 0) is also arbitrary, but avoids to convey any special meaning.